challenges/0-hello-world/java/tharindusoori/HelloWorld.java

```diff
@@ -0,0 +1,6 @@
+public class HelloWorld{
+    
+	public static void main(String[] args) {
```

Comment:
The block is not indented correctly.

Reply:
The closing was indented with spaces, but the opening of the block was indented with tab, making block appear not aligned.